Launcher changes

TheSpartaPT opened the pull request:
## SideBar / Settings:
Hide SideBar when in the settings view

## Login / Register Background:
Glass the whole background

## About Dialog:
Removed tooltips from the icon buttons, don't like how they looked, plus it's pretty obvious where each icon will lead you to
Replaced placeholder links

TheSpartaPT committed:
Hide SideBar when in the settings view

- Rerenders the sidebar component everytime the user navigates to a different url.

TheSpartaPT committed:
glass the whole background

TheSpartaPT committed:
About dialog changes

- Removed tooltips from the icon buttons, don't like how they looked, plus it's pretty obvious where each icon will lead you to
- Replaced placeholder links

seionmoya reviewed (changes requested):
Requested BG changes in private, the rest is good to merge!

TheSpartaPT committed:
Background Changes

- Made the background component more configurable.
- Added backgrounds to the game, index and settings view.
- reverted changes made to the login / register background.
